packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -471,7 +471,7 @@ function hasAnyConfiguration(provider: ProviderInfo) {
                 </Tooltip>
               </div>
               <div class="{container.status !== 'started' ? 'text-gray-900' : ''} font-semibold">
-                {container.name}
+                {container.displayName ?? container.name}
```

Comment:
probably make it mandatory on the info object (there will always be a displayName from renderer side)

but then on the server side, if displayName is missing from the definition we set it to the name

so, no need to handle any special case from the UI side, it's always here

Reply:
I made the info object have the displayName mandatory, and keep optional on the api for backward compatibility